package io.fortest.core.exception;

public class BadNumberBlockException extends ForTestException {

  public BadNumberBlockException() {
    super();
  }

  public BadNumberBlockException(String message) {
    super(message);
  }
}
